src/actions/toaster.js

```diff
@@ -0,0 +1,33 @@
+import actionTypes from '../constants/actions';
+
+/**
+ * An action to dispatch to display a toast
+ *
+ */
+export const toastDisplayed = data => ({
+  data,
+  type: actionTypes.toastDisplayed,
+});
+
+/**
+ * An action to dispatch to display a success toast
+ *
+ */
+export const successToastDisplayed = ({ type = 'success', ...rest }) =>
+  toastDisplayed({ type, ...rest });
+
+
+/**
+ * An action to dispatch to display an error toast
+ *
+ */
+export const errorToastDisplayed = ({ type = 'error', ...rest }) =>
+  toastDisplayed({ type, ...rest });
+
+/**
+ * An action to dispatch to hide a toast
+ *
+ */
+export const toastHidden = () => ({
```

Comment:
Toasts are by material design specs limited to one.

> "They should not be persistent or be stacked, as they are above other elements on screen."
> http://www.google.com/design/spec/components/snackbars-toasts.html#snackbars-toasts-usage

In Angular Material, it wasn't possible to stack them either.